Makefile

```diff
@@ -243,6 +243,11 @@ generate:
 	cd cmd/mdatagen && $(GOCMD) install .
 	$(MAKE) for-all CMD="$(GOCMD) generate ./..."
 
+.PHONY: mdatagen-test
+mdatagen-test:
```

Comment:
is this to generate mdatagen's data?  If so, should it live in cmd/mdatagen since it is specific to the component?

Reply:
We can but it's just pretty inconvenient. I want to have just one command that can be put in step 3 of the README Development section. I can put `cd ./cmd/mdatagen && (go run . metadata.yaml || cd -) && cd -` there but not sure it's a good one. Let me know what you think

Reply:
I guess it comes down to where we expect users to be while reading the development section.  If they are at "root" then `make mdatagen-test` will work but if they are in `cmd/mdatagen` it won't work right?

Reply:
It should be at the root since there is a final step 5 in the development process where you need to generate it for all the scrapers